examples/cpp/hello_world/main.cpp

```diff
@@ -1,25 +1,44 @@
 #include <iostream>
+#include <torch/script.h>
```

Comment:
on top of updating the example (which is great) I think we should also update the C++ API part of our README: https://github.com/pytorch/vision#c-api